debug/gdbserver.py

```diff
@@ -1015,6 +1015,32 @@ def test(self):
             if privilege in self.supported:
                 assertEqual(actual, privilege)
 
+class ComplianceTest(GdbTest):
+
+    def test(self):
+        output = self.gdb.command("monitor riscv test_compliance")
+        #TODO: Fix OpenOCD to remove the -rtos riscv requirement
+        assert(("ALL TESTS PASSED") in output or ("Please run with -rtos riscv to run compliance test.") in output)
+
+    def compile(self):
+        pass
+
+class SystemBusAccessConfigRegTest(GdbTest):
+
+    def test(self):
+        result = self.gdb.command("monitor riscv dmi_read 0x38")
+        if (int(result, base=16) == 0):
+            return 'not_applicable'
+        
```

Comment:
Can you make sure `make pylint` is happy before merging?

Reply:
sure, but can we add pylint check to the regression (because otherwise you assume I know how to run pylint...)

Reply:
pylint checks differ slightly between versions. I didn't put it in the regression to avoid the case where it works fine for a developer, but then fails a regression. The how-to-run pylint issue is why there's a `make pylint` target.